form_components/AppointmentForm.js

```diff
@@ -71,13 +73,21 @@ const styles = StyleSheet.create({
         marginBottom: 5,
         fontSize: 18,
         borderRadius: 6,
-        backgroundColor:'#EDEFEF'
+        backgroundColor:'#EDEFEF',
```

Comment:
Why are we hard-coding the value rather using GlobalColors?

```suggestion
        backgroundColor: GlobalColors.backgroundColor,
```

Reply:
It was more to see how it looked during dev, figured I would go back and use the stylesheet once we got the functionality down